### PR TITLE
EPMRPP-35547 React. LogItemActivity tab

### PR DESCRIPTION
**Pull request by ultd39**

### CODE REVIEW

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      newValue: PropTypes.string.isRequired,
+      field: PropTypes.string.isRequired,
+    }).isRequired,
+    projectRef: PropTypes.string.isRequired,
+  };
+
+  isValueEmpty = (value) => value === '';
+
+  renderAnalyzerIgnore = (value) =>
+    value === 'true' ? 'Игнорировать при Авто-Анализе | Ignore in Auto Analysis' : '';
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Please use localization

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    try {
+      const relevantItem = JSON.parse(value);
+      const itemLink = `#${projectRef}/launches/all/${relevantItem.launchId}/${Object.keys(
+        relevantItem.path_names,
+      ).join('/')}?log.item=${relevantItem.id}`;
+
+      return (
+        <Fragment>
+          {intl.formatMessage(messages.basedOn)}{' '}
+          <a className={cx('link')} href={itemLink}>
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Use `Link` from `redux-first-router`

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    } catch (error) {
+      return value;
+    }
+  }
+
+  renderTicketValue = (value) => {
+    const ticket = getTicketUrlId(value);
+
+    return ticket ? (
+      <a className={cx('link')} href={ticket.url} target="_blank">
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Use `Link from` `redux-first-router`

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx (hunk 1 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Ooops. Fixed.

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx (hunk 2 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Fixed.

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx (hunk 3 above)
Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Fixed.

**Comment by ultd39** [comment]
All issues resolved. And I also change base branch to `v5`.

File: app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        return value;
+    }
+  }
+
+  renderRelevantValue(value) {
+    const { projectRef, intl } = this.props;
+
+    try {
+      const relevantItem = JSON.parse(value);
+      const itemLink = `#${projectRef}/launches/all/${relevantItem.launchId}/${Object.keys(
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Please use objects for all links

Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: What do you mean?

Comment (DiscoElevator) [reply app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]:
```
const itemLink = {
  type: PROJECT_LOG_PAGE,
  payload: {
    projectId: projectRef,
    filterId, // can be obtained by using filterIdSelector (app/src/controllers/pages/selectors.js)
    testItemIds: [relevantItem.launchId, ...Object.keys(relevantItem.path_names), 
 relevantItem.id].join('/')
  }
};
```

Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemActivity/historyItem/historyItem.jsx]: Fixed.

File: app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemActivity/historyItem/historyItem.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  @media (max-width: $SCREEN_XS_MAX) {
+    width: auto;
+    padding-right: 0;
+    max-width: 50%;
+  }
+}
+
+.value {
+  color: $COLOR--charcoal-grey;
+  font: 11px/18px monospace;
```
Comment (DiscoElevator) [inline app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemActivity/historyItem/historyItem.scss]: font-family: $FONT-MENLO

Comment (ultd39) [reply app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemActivity/historyItem/historyItem.scss]: Fixed.

**Pull request merged**